cw/capture.py

```diff
@@ -991,6 +992,182 @@ def kmac_fvsr_key_batch(ctx: typer.Context,
     capture_end(ctx.obj.cfg)
 
 
+def capture_ecdsa_secret_keygen(ot, ktp, fw_bin, pll_frequency, capture_cfg):
```

Comment:
I suggest to rename this to something more generic, e.g. `capture_otbn_vertical()`. The reason for this is that `keygen` is just one/the first code snippet we agreed to do a vertical analysis for. In the future, we most likely will analyze also other snippets for which the settings will be very similar (but pretty different from analyzing full ECDSA). In that sense vertical analysis of OTBN code snippets is far closer to analyzing AES/KMAC than ECDSA. We could even do batch captures here but shouldn't need streaming. Capture rate should be 200 MHz.

 What might change for other snippets and we should try to keep things flexible in this regard:
- number of samples per trace (but they should be far below the max 132k samples)
- offsets
- number of inputs and length of inputs
- output length